frontends/main/src/page-components/AiRecommendationBot/AiRecommendationBotDrawerStrip.tsx

```diff
@@ -0,0 +1,208 @@
+  fill: theme.custom.colors.red,
+}))
+
+const Starters = styled.div(({ theme }) => ({
+  display: "flex",
+  gap: "16px",
+  maxWidth: "836px",
+  marginTop: "12px",
+  [theme.breakpoints.down("md")]: {
+    flexDirection: "column",
+  },
+}))
+
+const Starter = styled.div(({ theme }) => ({
+  flex: 1,
+  display: "flex",
+  alignItems: "center",
+  borderRadius: "8px",
+  border: `1px solid ${theme.custom.colors.lightGray2}`,
+  padding: "12px 16px",
+  color: theme.custom.colors.darkGray2,
+  [theme.breakpoints.down("md")]: {
+    textAlign: "center",
+    padding: "12px 36px",
+  },
+  ":hover": {
+    cursor: "pointer",
+    borderColor: "transparent",
+    color: theme.custom.colors.white,
+    backgroundColor: theme.custom.colors.darkGray1,
+  },
+}))
+
+const AiRecommendationBotDrawerStrip = () => {
+  const [open, setOpen] = useState(false)
+  const [initialPrompt, setInitialPrompt] = useState("")
+  const [showEntryScreen, setShowEntryScreen] = useState(true)
+  const aiChatRef = useRef<{
+    append: (message: Omit<AiChatMessage, "id">) => void
+  }>(null)
+
+  useEffect(() => {
+    if (!initialPrompt || showEntryScreen) return
+    const timer = setTimeout(() => {
+      aiChatRef.current?.append({
+        content: initialPrompt,
+        role: "user",
+      })
+      setInitialPrompt("")
+    }, 0)
+
+    return () => {
+      clearTimeout(timer)
+      setInitialPrompt("")
+    }
+  }, [initialPrompt, showEntryScreen])
+
+  const onPromptChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInitialPrompt(e.target.value)
+  }
+
+  const onPromptKeyDown: React.KeyboardEventHandler<HTMLInputElement> = (e) => {
+    if (e.key !== "Enter") return
+    setShowEntryScreen(false)
+  }
+
+  const onStarterClick = (content: string) => {
+    setInitialPrompt(content)
+    setShowEntryScreen(false)
+  }
+
+  const onDrawerClose = () => {
+    setOpen(false)
+    setShowEntryScreen(true)
+  }
+
+  return (
+    <StripContainer>
+      <DecorativeLine />
+      <LeadingText>Do you require assistance?</LeadingText>
+      <AskTIMButton onClick={() => setOpen(true)} />
+      <Drawer open={open} anchor="right" onClose={onDrawerClose}>
+        {showEntryScreen ? (
+          <EntryScreen>
+            <TimLogoBox>
+              <RiSparkling2Line />
+              <TimLogo src={timLogo.src} alt="" width={40} height={40} />
+            </TimLogoBox>
+            <Typography variant="h4">Welcome! I am TIM the Beaver.</Typography>
+            <Typography>Need assistance getting started?</Typography>
+            <StyledInput
+              fullWidth
+              size="chat"
+              onChange={onPromptChange}
+              onKeyDown={onPromptKeyDown}
+              endAdornment={
+                <AdornmentButton
+                  aria-label="Send"
+                  onClick={() => setShowEntryScreen(false)}
+                  disabled={!initialPrompt}
+                >
+                  <StyledSendButton />
```

Comment:
Suggest renaming this. It's not a button, and button inside button would have been invalid.